tests/shared/src/test/scala/cats/tests/OneAndSuite.scala

```diff
@@ -36,6 +48,47 @@ class OneAndSuite extends CatsSuite {
   implicit override val scalaCheckTestParameters: Parameters =
     Parameters.default.withMinSuccessfulTests(20).withMaxSize(Parameters.default.minSize + 5)
 
+  // Test kernel instances
+  {
+    import Helpers.Eqed
+
+    checkAll("OneAnd[F, A]", EqTests[OneAnd[ListWrapper, Eqed]].eqv)
```

Comment:
Were the `Eq` laws not already tested?

Reply:
I agree it may look superfluous (and probably it is). So if you don't like I am up to remove this check.

The motivation though was to give a try to the `Eqed` type that provides a "pure" `Eq` instance. So if somehow Eq is masked or overridden by Order or something and is not consistent with the laws, then it potentially could help to catch this. But I agree – it has a very little practical sense.

Also, I noticed that `Eqed` not in use in any other tests so decided to give it a try here.

Reply:
Oh, no, you misunderstood, I liked it!

I was just confirming that even though the `Eq` instance already existed, there were no existing law tests for it actually.